homeassistant/components/hvv_departures/.translations/en.json

```
@@ -0,0 +1,58 @@
+{
+    "config": {
+        "abort": {
+            "already_configured": "Device is already configured"
+        },
+        "error": {
+            "cannot_connect": "Failed to connect, please try again",
+            "cn_too_many": "Too many results. Please precise your query",
```

`Precise` is not a verb 😁
```suggestion
            "cn_too_many": "Too many results. Please state your query more precisely",
```

---

Please update this file as well.